### PR TITLE
Add Bowser.

### PR DESCRIPTION
**Pull request by OFFTKP**

### CODE REVIEW

**Review by wheremyfoodat — APPROVED** [review]

File: src/emulator.cpp [hunk 1]
```diff
@@ -202,6 +202,101 @@ std::filesystem::path Emulator::getAppDataRoot() {
...
+
+		// Update the has tried browser flag
+		FILE* file = fopen(getConfigPath() / "browser_tried", "w");
+		if (file) {
+			fclose(file);
+		}
+
+		// We don't need the emulator anymore, safely exit
+		#ifdef __x86_64__
+		volatile __asm__("hlt");
```
Comment (wheremyfoodat) [inline src/emulator.cpp:284]: I think this should crash using `mov [0], rax` as hlt will crash due to it being a privileged instruction instead :(

**Comment by OFFTKP** [comment]
Amazing catch !

File: src/emulator.cpp [hunk 2]
```diff
@@ -202,6 +202,101 @@ std::filesystem::path Emulator::getAppDataRoot() {
...
+		#else
+		Helpers::panic("Your architecture sucks :(");
+		#endif
+	} else {
+		// Inform users of our new web browser support!
+		system("echo 'Welcome to the Alber 3DS emulator! You can now browse the web with our new browser!'");
+		// Play a video that explains our new feature in great detail
+		system("start https://www.youtube.com/watch?v=ArCJ-9HJg0o");
```
Comment (wheremyfoodat) [inline src/emulator.cpp:292]: The app icon and name should be changed to properly reflect the values the emulator represents

File: src/emulator.cpp [hunk 3]
```diff
@@ -201,7 +201,106 @@ std::filesystem::path Emulator::getAppDataRoot() {
 	return appDataPath;
 }
 
+#ifdef __ANDROID
+, // I'm tired of supporting android, we shouldn't allow compilation to happen for it anymore
```
Comment (wheremyfoodat) [inline src/emulator.cpp:205]: Based

File: src/emulator.cpp [hunk 4]
```diff
@@ -201,7 +201,106 @@ std::filesystem::path Emulator::getAppDataRoot() {
...
+		path.filename() == "browser.smdh" ||
+		path.filename() == "browser.elf" ||
+		path.filename() == "browser.axf" ||
+		path.filename() == "browser.cci" ||
+		path.filename() == "browser.exe" ||
+		path.filename() == "browser.nds" ||
+		path.filename() == "browser.nes" ||
+		path.filename() == "browser.smc" ||
+		path.filename() == "browser.gb" ||
+		path.filename() == "browser.gba" ||
```
Comment (wheremyfoodat) [inline src/emulator.cpp:233]: Missing `browser.nds` as a 3DS can play DS games too

**Comment by OFFTKP** [comment]
fixed !

File: src/emulator.cpp [hunk 5]
```diff
@@ -201,7 +201,111 @@ std::filesystem::path Emulator::getAppDataRoot() {
...
+		system("start https://www.youtube.com/watch?v=ArCJ-9HJg0o");
+
+		// Make sure the user has tried it at least once before allowing them to move on to less important features
+		if (!hasTriedBrowser)
+			Helpers::panic("You must try the browser at least once before you can play games!");
+	}
+
+	// PEACHY SAID ADD APPROPRIATE ICON AND TITLE
+	// idk how to add icon, let's display an image instead when loading hte mulator
+	system("start https://imgur.com/E9jsNBc");
```
Comment (wheremyfoodat) [inline src/emulator.cpp:306]: I didn't know you were this sort of man

**Comment by wheremyfoodat** [comment]
It might be worth adding an anime maid of your preference to the docs folder for use in the emulator. I recommend watching this video to find some good material https://www.youtube.com/watch?v=irOiG61fu9w

**Comment by wheremyfoodat** [comment]
I can't wait to get that sweet money from the re:zero studio for promoting their series

**Comment by wheremyfoodat** [comment]
This was not meant to be posted here how do I delete a message

**Comment by OFFTKP** [comment]
grrr if youre gonna take this long to merge ill just delte e panda3ds :angry:

**Comment by wheremyfoodat** [comment]
Mean >:(

**Pull request closed without merging**